smart-contracts/contracts/PublicLock.sol~Stashed changes

```diff
@@ -0,0 +1,477 @@
+pragma solidity 0.4.24;
```

Comment:
hum, does that file belong here?

Reply:
Sorry, Just back from the dentist. hmm.  PublickLock.sol definitely belongs in this PR...let me check that out

Reply:
OK, just pushed to my origin. Let me review and see if that cleared up the "~Stashed changes"